hydra-cluster/test/Test/CardanoNodeSpec.hs

```diff
@@ -41,13 +41,13 @@ spec = do
               slot2 <- queryTipSlotNo networkId nodeSocket
               slot2 `shouldSatisfy` (> slot1)
 
-  it "withCardanoNodeOnKnownNetwork on mainnet starts synchronizing within 5 seconds" $
-    -- NOTE: This implies that withCardanoNodeOnKnownNetwork does not
+  it "withNewCardanoNodeOnKnownNetwork on mainnet starts synchronizing within 5 seconds" $
+    -- NOTE: This implies that withNewCardanoNodeOnKnownNetwork does not
     -- synchronize the whole chain before continuing.
     failAfter 5 $
       showLogsOnFailure "CardanoNodeSpec" $ \tr ->
         withTempDir "hydra-cluster" $ \tmp ->
-          withCardanoNodeOnKnownNetwork tr tmp Mainnet $
+          withNewCardanoNodeOnKnownNetwork tr tmp Mainnet $
```

Comment:
Could: have built in that check into the existing `withCardanoNodeOnKnownNetwork` and not need to change any names or interfaces at all!